src/ale.cpp

```diff
@@ -266,4 +254,115 @@ namespace ale {
    return result;
  }
 
+ std::string getPyTraceback() {
```

Comment:
This function essentially uses Python's built in traceback library to get a full stack trace of a py error.